packages/wrangler/src/cert/cert.ts

```diff
@@ -0,0 +1,199 @@
+import { createCommand, createNamespace } from "../core/create-command";
+import { requireAuth } from "../user";
+import { logger } from "../logger";
+import {
+	deleteMTlsCertificate,
+	getMTlsCertificate,
+	getMTlsCertificateByName,
+	listMTlsCertificates,
+	uploadMTlsCertificateFromFs,
+    uploadCaCertificateFromFs
+} from "../api/mtls-certificate";
+import type { MTlsCertificateResponse } from "../api/mtls-certificate";
+import { confirm } from "../dialogs";
+
+// wrangler cert
+export const certNamespace = createNamespace({
+	metadata: {
+		description: "🪪 Manage client mTLS certificates and CA certificate chains used for secured connections",
+		status: "open-beta",
+        owner: "Product: SSL",
+	},
+});
+
+export const certUploadNamespace = createNamespace({
+    metadata: {
+        description: "Upload a new cert",
+        status: "open-beta",
+        owner: "Product: SSL"
+    }
+})
+
+// wrangler cert upload mtls-certificate
+export const certUploadMtlsCommand = createCommand({
+	metadata: {
+		description: "Upload an mTLS certificate",
+		status: "stable",
+        owner: "Product: SSL"
+	},
+	args: {
+		cert: {
+			description: "The path to a certificate file (.pem) containing a chain of certificates to upload",
+			type: "string",
+			demandOption: true,
+		},
+		key: {
+			description: "The path to a file containing the private key for your leaf certificate",
+			type: "string",
+			demandOption: true,
+		},
+        name: {
+			description: "The name for the certificate",
+			type: "string"
+        }
+	},
+	async handler({ cert, key, name }, { config }) {
+        const accountId = await requireAuth(config);
+        logger.log(
+            name
+                ? `Uploading mTLS Certificate ${name}...`
+                : `Uploading mTLS Certificate...`
+        );
+        const certResponse = await uploadMTlsCertificateFromFs(accountId, {
+            certificateChainFilename: cert,
+            privateKeyFilename: key,
+            name,
+        });
+        const expiresOn = new Date(certResponse.expires_on).toLocaleDateString();
+        logger.log(
+            name
+                ? `Success! Uploaded mTLS Certificate ${name}`
+                : `Success! Uploaded mTLS Certificate`
+        );
+        logger.log(`ID: ${certResponse.id}`);
+        logger.log(`Issuer: ${certResponse.issuer}`);
+        logger.log(`Expires on ${expiresOn}`);
+    }
+});
+
+// wrangler cert upload mtls-certificate
```

Comment:
Need to update comment here.
```suggestion
// wrangler cert upload ca
```